### PR TITLE
CustomShortcutListBox: use event controllers

### PR DESCRIPTION
**Pull request by danirabbit**

### CODE REVIEW

File: src/Widgets/Shortcuts/CustomShortcutListBox.vala [hunk 1]
```diff
@@ -69,12 +69,15 @@ class Keyboard.Shortcuts.CustomShortcutListBox : Gtk.ListBox {
         public GLib.Settings gsettings { get; construct; }
         private bool is_editing_shortcut = false;
 
+        private Gtk.EventControllerKey key_controller;
+        private Gtk.GestureMultiPress keycap_controller;
+        private Gtk.GestureMultiPress status_controller;
```
Comment (jeremypw) [inline src/Widgets/Shortcuts/CustomShortcutListBox.vala:74]: It does not seem to be necessary to keep these references - you can just use `var`.

**Review by jeremypw — CHANGES_REQUESTED** [review]
When first opened, the custom shortcut displayed is the "default" one (Shift+Ctrl+Alt+F10) not the actual shortcut.  Also keeping references to the controllers seems unnecessary.

File: src/Widgets/Shortcuts/CustomShortcutListBox.vala [hunk 2]
```diff
@@ -398,11 +398,11 @@ class Keyboard.Shortcuts.CustomShortcutListBox : Gtk.ListBox {
 
             if (value_string != "") {
                 build_keycap_grid (value_string, ref keycap_grid);
-                keycap_stack.visible_child = keycap_eventbox;
+                keycap_stack.visible_child = keycap_stack;
```
Comment (jeremypw) [inline src/Widgets/Shortcuts/CustomShortcutListBox.vala:401]:
```suggestion
                keycap_stack.visible_child = keycap_grid;
```

A stack cannot be its own child!

Comment (danirabbit) [reply src/Widgets/Shortcuts/CustomShortcutListBox.vala:401]: Woops, thank you. I thought I fixed this :sweat_smile:

File: src/Widgets/Shortcuts/CustomShortcutListBox.vala (hunk 1 above)
Comment (danirabbit) [reply src/Widgets/Shortcuts/CustomShortcutListBox.vala:74]: In my testing the controllers don't seem to work at all unless we hold a ref to them in GTK 3

Comment (jeremypw) [reply src/Widgets/Shortcuts/CustomShortcutListBox.vala:74]: I'll check again - I may have only tested for compilability.

**Pull request merged**